core/cap-0038.md

```diff
@@ -648,6 +652,9 @@ specified by minPrice and maxPrice.
 The process of applying `LiquidityPoolDepositOp lpdo` with source `sourceAccount`
 is
 ```
+if (ledgerHeader.v1.flags & DISABLE_LIQUIDITY_POOL_DEPOSIT_FLAG) == DISABLE_LIQUIDITY_POOL_DEPOSIT_FLAG
```

Comment:
Same thing about moving this to validation time. Same applies for withdraw.

Reply:
Done